contributing/index.md

```diff
@@ -221,7 +219,7 @@ currently an error for the HEAD revision of WebRTC when built inside Chromium.
 
 ## Committing your CL
 
-After the review process is done and you get LGTM (Looks Good To Me) from all
+After the review process is done and you get CR+1 (Code-Review +1) from all
 reviewers you can go ahead and submit your change, assuming you're an approved
 committer. If you're not a committer, you'll need to ask one of the reviewers
```

Comment:
> If you're not a committer, you'll need to ask one of the reviewers to submit the CL for you using the Commit Queue (CQ).

Not sure if this is still true

Reply:
Hmm, I don't think it ever was. As long as you got the lgtms from committers (and owners) I think you can check the CQ yourself. But I'm not sure, it may actually not be possible. Maybe ask someone from Chrome infra - Andrii should know.